internal/osbuildconfig/osbuildconfig_handler.go

```diff
@@ -0,0 +1,96 @@
+package osbuildconfig
+
+import (
+	"fmt"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/project-flotta/osbuild-operator/internal/httpapi"
+	"github.com/project-flotta/osbuild-operator/internal/repository/configmap"
+	"github.com/project-flotta/osbuild-operator/internal/repository/osbuildconfigtemplate"
+	"net/http"
+
+	loggerutil "github.com/project-flotta/osbuild-operator/internal/logger"
+	"github.com/project-flotta/osbuild-operator/internal/manifests"
+	repositoryosbuildconfig "github.com/project-flotta/osbuild-operator/internal/repository/osbuildconfig"
+	"github.com/project-flotta/osbuild-operator/internal/repository/secret"
+	"github.com/project-flotta/osbuild-operator/restapi"
+	"k8s.io/apimachinery/pkg/api/errors"
+)
+
+var Config httpapi.Config
+
+type OSBuildConfigHandler struct {
+	OSBuildConfigRepository         repositoryosbuildconfig.Repository
+	SecretRepository                secret.Repository
+	OSBuildCRCreator                manifests.OSBuildCRCreator
+	OSBuildConfigTemplateRepository osbuildconfigtemplate.Repository
```

Comment:
Please remove `OSBuildConfigTemplateRepository` and `ConfigMapRepository` - they are unused.